client/src/components/Workflow/WorkflowActionsExtend.vue

```diff
@@ -72,11 +65,41 @@ async function onRestore() {
         Toast.info("Workflow restored");
     }
 }
+
+const root = computed(() => {
+    const port = window.location.port ? `:${window.location.port}` : "";
```

Comment:
If you search for `window.location.port` you find other instances of this, maybe you could push this to a shared helper function ?

Reply:
I created a `getFullAppUrl` util function and replaced all instances of `window.location.port` with it where needed.